au4/src/au3wrap/internal/au3selectioncontroller.cpp

```diff
@@ -0,0 +1,40 @@
+#include "au3selectioncontroller.h"
+
+using namespace au::au3;
+
+void Au3SelectionController::resetDataSelection()
+{
+    m_selectedTrackIds.set(std::vector<processing::TrackId>());
+    m_selectedStartTime.set(-1.0);
+    m_selectedEndTime.set(-1.0);
+}
+
+muse::ValCh<std::vector<au::processing::TrackId> > Au3SelectionController::dataSelectedOnTracks() const
```

Comment:
in this module (au3) it is problematic to use usings, because here two domain models intersect (even some type names coincide, for example TrackId), we need to clearly see which type, from which model, is meant here.